man/man5/zfs-module-parameters.5

```diff
@@ -753,6 +753,17 @@ Disable pool import at module load by ignoring the cache file (typically \fB/etc
 Use \fB1\fR for yes (default) and \fB0\fR for no.
 .RE
 
+.sp
+.ne 2
+.na
+\fBzfs_checksums_per_second\fR (int)
```

Comment:
I think we should note that setting these values too low could cause the zed to not behave as expected.

Reply:
For posterity...
As I read the code, `checksum_N` = 10 and `checksum_T` = 10 minutes, not 1 second in the SERD engine. This is consistent with Solaris. So effectively, we're good unless set to 0, which is effectively disabling the ereports.